[2024-04-17T05:20:39.323+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-17T05:20:39.347+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T10:32:00+00:00 [queued]>
[2024-04-17T05:20:39.356+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T10:32:00+00:00 [queued]>
[2024-04-17T05:20:39.357+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-17T05:20:39.373+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): hello> on 2024-04-16 10:32:00+00:00
[2024-04-17T05:20:39.385+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=62) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-17T05:20:39.387+0000] {standard_task_runner.py:63} INFO - Started process 90 to run task
[2024-04-17T05:20:39.388+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'spotify_final_dag', 'hello', 'scheduled__2024-04-16T10:32:00+00:00', '--job-id', '1567', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpbuyg22lb']
[2024-04-17T05:20:39.390+0000] {standard_task_runner.py:91} INFO - Job 1567: Subtask hello
[2024-04-17T05:20:39.474+0000] {task_command.py:426} INFO - Running <TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T10:32:00+00:00 [running]> on host 52651c1f9987
[2024-04-17T05:20:39.618+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spotify_final_dag' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T10:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-16T10:32:00+00:00'
[2024-04-17T05:20:39.621+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-17T05:20:43.366+0000] {logging_mixin.py:188} INFO - There was a problem with the connection. Trial number 1, Status Code: 401
[2024-04-17T05:20:46.659+0000] {logging_mixin.py:188} INFO - There was a problem with the connection. Trial number 2, Status Code: 401
[2024-04-17T05:20:46.663+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-17T05:20:46.665+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-17T05:20:46.680+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=spotify_final_dag, task_id=hello, execution_date=20240416T103200, start_date=20240417T052039, end_date=20240417T052046
[2024-04-17T05:20:46.748+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-17T05:20:46.782+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-17T05:20:46.788+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
